db/models.py

```diff
@@ -50,3 +60,53 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
```

Comment:
Code Style Issue: According to the checklist, you should define `class Meta` after all attributes are defined. Please move `class Meta` after the `user` attribute.